AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/Base Classes/ALHomeViewController.swift

```diff
@@ -13,7 +13,7 @@ import SafariServices
 
 class ALHomeViewController: UITableViewController
 {
-    let kSupportLink = "https://support.applovin.com/support/home"
+    let kSupportLink = "https://support.applovin.com/hc/en-us"
     
     let kRowIndexToHideForPhones = 3;
```

Comment:
nit: But I would remove the newline between the 2 variables to match Obj-C

Reply:
Even more nit, I'd remove the unnecessary `;` :)

Reply:
added